comfy_cli/cancellation.py

```
@@ -0,0 +1,117 @@
+        if callable(previous) and previous not in (signal.SIG_DFL, signal.SIG_IGN):
+            previous(signum, frame)
+        else:
+            # Default: raise KeyboardInterrupt in the main thread.
+            raise KeyboardInterrupt()
+
+    signal.signal(signal.SIGINT, _handler)
+    _INSTALLED = True
+    return token
+
+
+def reset_for_testing() -> None:
```

Pylint (4.0.5)

[convention] Missing function or method docstring

(C0116)